lib/OA/Creative/File.php

```diff
@@ -155,7 +154,7 @@ function &factory($filePath, $fileName = '')
         $oCreative = new $className($fileName);
 
         $result = $oCreative->loadFile($filePath);
-        if (PEAR::isError($result)) {
+        if ((new PEAR)->isError($result)) {
```

Comment:
Hmm, there might be lots of `PEAR::isError`, I don't think we want to create an instance of PEAR every time.

Reply:
This has serious implications in future PHP versions. 
This type of calls are deprecated in 7.0 (https://www.php.net/manual/en/migration70.deprecated.php#migration70.deprecated.static-calls), probably removed in v8.